### DISCUSSION TITLE
verification mail in Spam

### DISCUSSION THREAD

**Discussion by Skyfay** [dani-garcia/vaultwarden#1856] [Q&A]
I noticed that the verification emails always end up in spam.
The parameters should possibly be adjusted so that they are not declared as spam.

**Comment by BlackDex** [comment]
There are multiple reasons why an email could be marked as spam. So this report isn't going to help us. We need to know why it was triggered. 
Also, spf, dkim and dmarc configurations can play an issue here which is out of the control of Vaultwarden

**Comment by Masgalor** [comment]
If you want to better understand why your mails get marked as spam I would recommend to use [Mail-Tester](https://www.mail-tester.com/) or a similar service. They will give you a detailed report about why your mails arent trustworthy enough.

**Comment by Skyfay** [comment]
![image](https://user-images.githubusercontent.com/76796007/126158422-3dc0ffa4-6441-46fe-9c3f-a6441f105ed5.png)
![image](https://user-images.githubusercontent.com/76796007/126158472-07b2e9b9-cf31-4491-b54a-e20aa3c004ea.png)
![image](https://user-images.githubusercontent.com/76796007/126158488-7bdf4429-7e1d-4858-850c-ba717b0340c1.png)
The rest is good everywhere

**Comment by BlackDex** [comment]
@Masgalor I used that already, and i have think i have fixed all issues.
But since i do not know what type of spam filtering the reporter is using and that mail-tester only uses the default spamassissin rules, it's hard to tell what triggered it.

**Comment by BlackDex** [comment]
@Skyfay, it looks to me it's marked as spam because of the items i mentioned which are out of control of vaultwarden.
SPF, DKIM and DMARC.

**Comment by Skyfay** [comment]
@BlackDex So is the problem with Bitwarden or my provider?

**Comment by Masgalor** [comment]
@BlackDex my comment was mostly meant for Skyfy because I agree with you that the problem is most likely not related to vaultwarden.

@Skyfay the quote "SpamAssassin mag deine E-Mail" (SpamAssassin likes your E-Mail) indicates that the content of the mail is not suspicious. But those entries with a negativ number in the score are missing PTR-Record, invalid SPF-Record and missing DKIM-signature. All three of them are problems with mailserver not vaultwarden.

I assume most mail server would classify this mail as spam, for example my mine would definitly mark it as spam.

**Comment by Skyfay** [comment]
@Masgalor I added the SPF record and now it no longer ends up in spam!
thanks
